README.md

```diff
@@ -449,13 +449,13 @@ You can use Clarinet to publish your contracts to Devnet / Testnet / Mainnet env
 The first step is to generate a deployment plan, with the following command: 
 
 ```bash
-$ clarinet deployment generate --mainnet
+$ clarinet deployments generate --mainnet
 ```
 
 After **cautiously** reviewing (and updating if needed) the generated plan, you can use the command:
 
 ```bash
-$ clarinet deployment apply -p <path-to-plan.yaml>
+$ clarinet deployments apply -p <path-to-plan.yaml>
```

Comment:
Thanks @mcintyre94!
How would you feel about fixing the line
https://github.com/hirosystems/clarinet/blob/a8e746096dfda411d96627f82c500c737ddf149b/src/frontend/cli.rs#L108
instead?
In `aliases`, we should have `deployment` instead of `deployments`. Supporting both singular and plural is just more convenient in terms of DX.

Reply:
Ah, that's what was going on! The "Did you mean 'deployments' or 'deployments'?" message should have clued me in :) Much better fix, updated the PR - TY!

Reply:
Awesome, thank you @mcintyre94!